src/python/dxpy/scripts/dx.py

```diff
@@ -2409,6 +2409,12 @@ def close(args):
 
 def wait(args):
     had_error = False
+    # If only one path provided, check to see if it is a local file
+    # and if so gather actual paths on which to wait from the contents 
+    # of the file.
+    if len(args.path) == 1 and os.path.isfile(args.path[0]):
+        args.path = open(args.path[0]).read().strip().split('\n')
```

Comment:
Please, try/catch here in case the file cannot be read due to insufficient permissions, etc. You can catch IOError like in this case: https://github.com/dnanexus/dx-toolkit/blob/c9135627a772a7a51c09624f433a39d6188250d1/src/python/dxpy/scripts/dx_build_app.py#L330-L334 and raise DXCLIError.